### PR TITLE
Content

### PR DESCRIPTION
**Pull request by Incorrect-password**
I have added content or filler content for all of my pages. These are not formatted. I have also beutified teh nav a touch.

### CODE REVIEW

File: homepage.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<link href="stylesheetportfolio.css" rel="stylesheet" type="text/css">
 		<title>JVT</title>
 	</head>
-	<body>
+	<body class = "homePage">
```
Comment (mporam) [inline homepage.html]: spaces!

File: aboutme.html [hunk 2]
```diff
@@ -23,17 +23,17 @@
...
 				</div>
 				
-				<div class="flexContainer contacts">
-					<p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam optio repudiandae ipsam animi voluptate omnis enim aspernatur rem deleniti eveniet nam molestias vel deserunt vitae sapiente iste mollitia, eos corrupti.</p>
+				<div class="flexContainer contact">
+					<p> If you would like to contact me for any reason including to arrange a phone call please fill out the form below and I will get back to you as soon as possible.</p>
 					<form>
-						<input type="text-area" name="emailBody">
-						<input type="submit" value="Submit">
+							<textarea name="emailBody" form="email" placeholder ="What can I help you with?" class="form"></textarea>
```
Comment (mporam) [inline aboutme.html]: why? fix indentation

File: homepage.html [hunk 3]
```diff
@@ -23,20 +23,26 @@
...
 	
 		<div class="container">
-		
-			<div class="floatingText">
-			</div>
 			
-			<div class="mediaContainer">
+			<div>
+				<div class="textBox">
+					<p class="frontPage Header">James Taylor</p>
```
Comment (mporam) [inline homepage.html]: try and keep your classnames starting with lowercase letters

File: homepage.html [hunk 4]
```diff
@@ -23,20 +23,26 @@
...
+				</div>
 				
-				<div class="mediaIcon">
-				</div>	
-				<div class="mediaIcon">
+				<div class="textBox">
+					<p class="frontPage Sub">Full Stack Web Developer</p>
```
Comment (mporam) [inline homepage.html]: lowercase classes

File: stylesheetportfolio.css [hunk 5]
```diff
@@ -14,28 +14,50 @@ body {
 	justify-content: space-evenly;
 }
 
+.navLink {
+	padding: 10px 20px 0px 20px;
+	background: ;
```
Comment (mporam) [inline stylesheetportfolio.css]: this does nothing, please remove

File: stylesheetportfolio.css [hunk 6]
```diff
@@ -14,28 +14,50 @@ body {
 	justify-content: space-evenly;
 }
 
+.navLink {
+	padding: 10px 20px 0px 20px;
```
Comment (mporam) [inline stylesheetportfolio.css]: 0 values dont need a unit - 0 of anything is 0

File: aboutme.html [hunk 7]
```diff
@@ -7,8 +7,8 @@
 		<link href="stylesheetportfolio.css" rel="stylesheet" type="text/css">
 		<title>Profile</title>
 	</head>
-		<body>
-			<nav class= "container">
+		<body class="aboutMe">
```
Comment (ashleycoles) [inline aboutme.html]: Why the class on the body? There should never be two body tags so there shouldn't be any need for a class on it.

File: homepage.html [hunk 8]
```diff
@@ -7,8 +7,8 @@
 		<link href="stylesheetportfolio.css" rel="stylesheet" type="text/css">
 		<title>JVT</title>
 	</head>
-	<body>
-		<nav class= "container">
+	<body class="homePage">
```
Comment (ashleycoles) [inline homepage.html]: Again, is there any need for the class on the body?

File: homepage.html [hunk 9]
```diff
@@ -23,20 +23,25 @@
...
-				<div class="mediaIcon">
-				</div>	
-				<div class="mediaIcon">
+				<div class="textBox">
+					<p class="frontPageSub">Full Stack Web Developer</p>
 				</div>
-				<div class="mediaIcon">
+				
+				<div class="mediaContainer">
+				</div>
```
Comment (ashleycoles) [inline homepage.html]: Should this closing tag be here?

File: projects.html [hunk 10]
```diff
@@ -7,8 +7,8 @@
 		<link href="stylesheetportfolio.css" rel="stylesheet" type="text/css">
 		<title>Projects</title>
 	</head>
-		<body>
-			<nav class= "container projectContainer">
+		<body class="projects">
```
Comment (ashleycoles) [inline projects.html]: Any need for the class on the body?

File: stylesheetportfolio.css [hunk 11]
```diff
@@ -9,45 +9,132 @@ html {
...
+	float: right;
 	display: flex;
-	justify-content: center;
 	flex-wrap: wrap;
-	padding:0px;
+	padding: 0px;
+}
+.form {
+	height:60vh;
+	width:25vw;
```
Comment (ashleycoles) [inline stylesheetportfolio.css]: These two rules don't match the formatting of the rest, they should have a space after the colon.

File: stylesheetportfolio.css [hunk 12]
```diff
@@ -57,16 +144,17 @@ img {
...
-.textBox { 
-	align-content: center;
-	justify-content: center;
-	width: 50%;
-	padding: 6%
-}
+
+
+
+
```
Comment (ashleycoles) [inline stylesheetportfolio.css]: Remove all of this whitespace

File: homepage.html (hunk 9 above)
Comment (Incorrect-password) [reply homepage.html]: no

**Pull request merged**